ksql-engine/src/main/java/io/confluent/ksql/function/udf/math/Abs.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"; you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.function.udf.math;
+
+import io.confluent.ksql.function.udf.Udf;
+import io.confluent.ksql.function.udf.UdfDescription;
+import io.confluent.ksql.function.udf.UdfParameter;
+import io.confluent.ksql.function.udf.UdfSchemaProvider;
+import io.confluent.ksql.util.DecimalUtil;
+import java.math.BigDecimal;
+import java.util.List;
+import org.apache.kafka.connect.data.Schema;
+
+@UdfDescription(name = "Abs", description = Abs.DESCRIPTION)
+public class Abs {
```

Comment:
instead of deleting, we should update `AbsKudfTest` to add tests for BigDecimal (and rename it to `AbsTest`)